documentation/Using-the-SDK/Creating-an-extended-table.md

```diff
@@ -42,34 +43,48 @@ This class __MUST__ have the following items:
                     // Paths to the Cookers that are needed for
                     // this table. This is not needed if you are
                     // using the RequiresCooker attributes.
-                    //
                 });
             );
 
-        // This method is required so that the runtime can build your
-        // table.
+        //
+        // This method, with this exact signature, is required so that the runtime can 
+        // build your table once all cookers have processed their data.
         //
         public static void BuildTable(
             ITableBuilder tableBuilder,
-            IDataExtensionRetrieval tableData
+            IDataExtensionRetrieval requiredData
         )
         {
-            // Query cooked data from dataRetrieval, and use the
+            //
+            // Query cooked data from requiredData, and use the
             // tableBuilder to build the table.
             //
         }
     }
 ````
 
-There is no restriction on the Cookers on which your Table may depend. You
-may author your own Cookers leveraging Cookers from many different plugins,
-and have your Table depend on that. As long as you have Cooked Data, you
-can create an Extended Table.
+There is no restriction on the cookers on which your `Table` may depend. You
+may author your own cookers leveraging cookers from many different plugins,
+and have your `Table` depend on that. As long as you have cooked data, you
+can create an extended table.
 
-As long as the DLL with your Extended Tables is available to WPA or the
-Engine, then your Table will be available for use.
+As long as the DLLs that contain
```

Comment:
Are DLLs known pretty well to our reader? Maybe clarify that this is a .dll file?

Reply:
The reader _should_ know what a DLL is, but I edited this to avoid it entirely